### DISCUSSION TITLE
Security Vulnerabilities

### DISCUSSION THREAD

**Discussion by accessgames** [Command-Enterprises/Commander#15] [Report Issues]
If anybody sees security issues, you can reply here to report it.

**Comment by Quartinal** [comment]
@accessgames, there seems to be a **critical** issue [here](https://github.com/Command-Enterprises/Commander/security/code-scanning/1). Is this because of one of your commits? It seems we will have XSS attacks very soon, so we will need to focus on closing this one (see [server-side request forgery](https://cwe.mitre.org/data/definitions/918.html) also).
